src/app/ubs/ubs-user/ubs-user-orders-list/ubs-user-order-payment-pop-up/ubs-user-order-payment-pop-up.component.ts

```diff
@@ -96,13 +98,27 @@ export class UbsUserOrderPaymentPopUpComponent implements OnInit {
   }
 
   public certificateSubmit(index: number, certificate: FormControl): void {
-    if (!this.userCertificate.certificates.includes(this.formArrayCertificates.value[index])) {
-      this.userCertificate.certificates.push(this.formArrayCertificates.value[index]);
+    this.isCertBeenUsed = false;
+    if (
+      !this.usedCertificates ||
+      !this.usedCertificates.some((item, ind) => {
+        return item === certificate.value.certificateCode && ind !== index;
```

Comment:
don't use return here
!this.usedCertificates.some((item, ind) => item === certificate.value.certificateCode && ind !== index)